Sources/PubNubChat/PubNub/Models/ChatMessage.swift

```diff
@@ -74,8 +74,8 @@ public struct ChatMessage<Custom: ChatCustomData>: Identifiable, Hashable, Codab
       id = try container.decode(String.self, forKey: .id)
       text = try container.decode(String.self, forKey: .text)
       contentType = try container.decodeIfPresent(String.self, forKey: .contentType)
-      content = try container.decodeIfPresent(AnyJSON.self, forKey: .content)
-      custom = try container.decodeIfPresent(Custom.Message.self, forKey: .custom) ?? Custom.Message()
+      content = try container.decodeIfPresent(Custom.Message.Content.self, forKey: .content)
+      custom = try container.decodeIfPresent(Custom.Message.Custom.self, forKey: .custom) ?? Custom.Message.Custom()
```

Comment:
this looks suspicious to me, maybe `custom` shouldn't be optional, in that case this init could be not necessary.

Reply:
It looks like `content` and `custom` are [required](https://github.com/pubnub/chat-specifications/blob/master/Specifications/00001%20-%20Entity%20Models.md#message-content) across chat platforms, but we'll want to validate with other platforms what an "empty object" serializes into before we remove this safety net.

Reply:
Yeah, let's validate it with other platforms. I think that  `id`, `text`and `createdAt` are required only - see the `required` key in [this](https://github.com/pubnub/chat-specifications/blob/master/Specifications/00001%20-%20Entity%20Models.md#message-content) JSON schema you also sent me above. I'll bring up this topic because I've got the impression that we may have differences between platforms

Reply:
Yeah I don't know what was wrong with me, but I completely misread that doc.  :D  Thank you for confirming though!